src/sentry/lang/javascript/processor.py

```diff
@@ -578,15 +614,16 @@ def expand_frames(self, frames, release):
                     })
 
                 # Store original data in annotation
+                # HACK(dcramer): we stuff things into raw which gets popped off
+                # later when adding the raw_stacktrace attribute.
+                raw_frame = frame.to_json()
                 frame.data = {
-                    'orig_lineno': frame.lineno,
```

Comment:
I changed it to this:

![image](https://cloud.githubusercontent.com/assets/2153/15621403/7a53a854-2415-11e6-8526-6f424c93bdf8.png)

It now pops on hover, and the URL is no longer clickable (clickable is useless for uploaded map files anyways). We also now show the full URL, since that's more helpful for debugging.

Not being able to copy/paste anymore (since the tooltip is on hover) *might* be a drag. I'm not sure. We can push this and find out.